src/styles/components/DataGridWithInfinteScroll.scss

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+@import "../common/colors";
+@import "../common/fonts";
+@import "../common/borders";
+
+$csg-row-height: 48px;
+$csg-header-height: 48px;
+$table-border: 1px solid $gray-300;
+$csg-table-th-padding: 0.563rem;
+
+.data-grid-inifnite-table {
+  width: 100%;
+  flex: 1;
+  border: $table-border;
+
+  thead {
+    border: $table-border;
+    border-bottom: unset;
+  }
+
+  tbody {
+    border: $table-border;
+
+    tr {
+      &:hover {
+        background-color: $gray-100;
+      }
+
+      &:focus {
+        background-color: $blue-50;
+      }
+
+      &:active {
+        background-color: $blue-100;
+      }
+    }
+  }
+
+  tr {
+    border-top: $table-border;
+  }
+
+  th {
+    position: relative;
+    font-size: 14px;
```

Comment:
Font sizes should be applied at the document level, not at every component level.  If you really need this, use the font-size variables

Reply:
Done